openpype/hosts/maya/api/lib.py

```diff
@@ -2097,13 +2102,11 @@ def reset_frame_range():
     if handle_end is None:
         handle_end = handles
 
-    frame_start -= int(handle_start)
-    frame_end += int(handle_end)
+    animation_start = frame_start - int(handle_start)
+    animation_end = frame_end + int(handle_end)
```

Comment:
Similar to the line above. Isn't `frame_end += int(handle_end)` equal to `frame_end = frame_end + int(handle_end`)?